cms/static/js/certificates/collections/certificates.js

```diff
@@ -0,0 +1,41 @@
+// Backbone.js Application Collection: Certificates
+
+define([
+    'backbone', 'js/certificates/models/certificate'
+],
+function(Backbone, Certificate) {
+    'use strict';
+    var CertificateCollection = Backbone.Collection.extend({
+        model: Certificate,
+
+        initialize: function(attr, options) {
+            // Set up the attributes for this collection instance
+            this.url = options.certificateUrl;
+        },
+
+        IsJsonString: function(str) {
+            // Validates the format of the provided string
+            try {
+                JSON.parse(str);
+            } catch (e) {
+                return false;
+            }
+            return true;
+        },
+
+        parse: function (certificatesJson) {
+            // Transforms the provided JSON into a Certificates collection
+            var modelArray;
+            if(this.IsJsonString(certificatesJson)) {
```

Comment:
This approach means that you're calling `JSON.parse` twice on any string - why not test if the value is a string - if a string, then `JSON.parse`, otherwise assume it's already a json object.

Alternatively, could have the translation / non-translation happen inside some wrapped function - e.g.
```
certificate_array: function(certificate_info) {
    var return_array;
    try {
        return_array = JSON.parse(certificate_info);
    } catch(e) {
        return_array = certificate_info;
    }
    return return_array;
}
```